vignettes/user.Rmd

```diff
@@ -0,0 +1,73 @@
+---
+title: "Scripts"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Scripts}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+options("lifecycle:::calling_package" = "tibble")
+```
+
+The lifecycle package uses warnings to advertise which functions are deprecated.
+Deprecated functions will be removed in a future release, so it's good practice to eliminate the warnings as soon as you see them.
+
+```{r setup}
+library(lifecycle)
+```
+
+For example, lets imagine your code used `tibble::data_frame()`, which was deprecated in favour of `tibble()` in version 1.1.0.
+`data_frame()` now looks something like this:
+
+```{r}
+data_frame <- function(...) {
+  deprecate_warn("1.1.0", "data_frame()", "tibble()")
+  tibble::tibble(...)
+}
+```
+
+So if you use `data_frame()` in your own code you'll get a warning:
+
+```{r}
+df1 <- data_frame(x = 1, y = 2)
+df2 <- data_frame(a = "apple", b = "banana")
+```
+
+You'll notice that the warning only appears the first time we call it --- lifecycle only notifies you every 8 hours to avoid driving you mad if you've used a deprecated function in many places.
```

Comment:
```suggestion
You'll notice that the warning only appears the first time we call it --- lifecycle only notifies you every 8 hours to avoid disrupting your work if you've used a deprecated function in many places.
```